### PR TITLE
Allow make_fe_url to create url from null value of frontend_url

### LINKED ISSUE #4137: Allow make_fe_url to generate url even if frontend_url setting is null
**I'm submitting a ...**  (check one with "x")
- [x] bug report
- [ ] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-orga-server

open-event-api user registration is not working and I guess this is the most probable reason for it.

### PR DESCRIPTION
**Pull request by poush**
Fixes #4137 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `nextgen` branch.
- [x] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
Let make_fe_url not break the process flow for null fields.

### CODE REVIEW

File: app/api/helpers/files.py [hunk 1]
```diff
@@ -169,5 +169,5 @@ def make_fe_url(settings=None, path=None):
     """
     if settings is None:
         settings = get_settings()
-    fe_host = settings['frontend_url']
+    fe_host = settings['frontend_url'] if settings['frontend_url'] else ""
```
Comment (SaptakS) [inline app/api/helpers/files.py]: fe?

Comment (poush) [inline app/api/helpers/files.py]: front-end-host => fe_host 😅

Comment (SaptakS) [inline app/api/helpers/files.py]: @poush then name it front_end_host only I guess. 😅

Comment (poush) [inline app/api/helpers/files.py]: Done :)

**Pull request merged**